app/src/main/java/net/aiscope/gdd_app/extensions/Snackbar.kt

```diff
@@ -0,0 +1,44 @@
+package net.aiscope.gdd_app.extensions
+
+import android.content.Context
+import android.text.SpannableStringBuilder
+import android.text.style.ImageSpan
+import android.widget.TextView
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.core.text.color
+import androidx.core.text.scale
+import com.google.android.material.snackbar.BaseTransientBottomBar
+import com.google.android.material.snackbar.Snackbar
+import net.aiscope.gdd_app.R
+
+const val SNACKBAR_TEXT_SCALE = 1.2f
+
+fun Snackbar.setActionOnDismissed(action: () -> Unit): Snackbar {
```

Comment:
What is the reason to create this extension methods? after review the code, maybe is a better idea build a custom component instead to use the android component?

Reply:
Extension methods removed. A custom component has been created.